lib/styles/trackInputModality.js

```diff
@@ -76,7 +76,9 @@ function setUpEventHandlers() {
     return triggers;
   };
 
-  disableFocusRingByDefault();
+  if (disableDefaultFocusRing) {
+    disableFocusRingByDefault();
+  }
```

Comment:
I'm a bit confused by the naming and intent here: "disable default focus ring" and "focus ring disabled by default" seem like two different concepts.

Reply:
Thank you, that's a language nuance I haven't recognized. I just need to find a name for the newly added parameter that would say:
If `<parameter>` is `true`, call `disableFocusRingByDefault`
If `<parameter>` is `false`, don't call `disableFocusRingByDefault`

Do you have any ideas for naming that's easy to understand?

Reply:
I actually don't understand why this has a separate function defined inline. I think it could just be:

```js
function setUpEventHandlers(disableFocusRingByDefault) {

  // ...

  if (disableFocusRingByDefault) {
    const css = 'body:not([modality=keyboard]) :focus { outline: none; }';
    const head = document.head || document.getElementsByTagName('head')[0];
    const style = document.createElement('style');

    style.type = 'text/css';
    style.id = 'disable-focus-ring';

    if (style.styleSheet) {
      style.styleSheet.cssText = css;
    } else {
      style.appendChild(document.createTextNode(css));
    }

    head.insertBefore(style, head.firstChild);
  }

  // ...
}
```

and the

Reply:
That's a good point, I think the only reason might be that it was copied from the original library in this way.